index.html

```diff
@@ -2355,13 +2355,23 @@ <h3>Unlinkability</h3>
       <section>
         <h3>Selective Disclosure</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are selective
-disclosure digital signature schemes, such as BBS+, that are capable of
-not disclosing correlatable identifiers and ensuring that a different
-but valid digital signature is re-created upon every presentation.
+        <p>
+Selective disclosure is a technique that enables the sender of a pre-signed
+message to reveal parts of the message without revealing the entire message. For
+example, selectively disclosing a digital driver's license for the purposes of
+renting a car might involve the entity presenting the license to only reveal the
+issuing authority, license number, birthday, and authorized motor vehicle class.
+Note that in this case, the license number is correlatable information that is
```

Comment:
the correlation message is important. I know there are selective disclosure schemes which are correlation-resistant, such as https://github.com/decentralized-identity/crypto-wg/blob/main/work_items/spartan_zkSNARK_signatures.md -- maybe some language noting that this capability is implementation specific and correlation is a separate problem to solve for?

Reply:
@decentralgabe I added language pointing to the section on unlinkability to address your concern in 355186b70de9a58cd14d120c94e73cd1607e42c0.

Reply:
@decentralgabe I added language pointing to the section on unlinkability to address your concern in 355186b70de9a58cd14d120c94e73cd1607e42c0.

Reply:
@mprorock added your example here: f5fd46960051f8ba6ce8d89bb3527aa4245e3dab.